terraform/heroku.tf

```diff
@@ -25,6 +25,7 @@ module "django" {
 
   additional_django_vars = {
     DJANGO_API_URL    = "https://app.shapeworks-cloud.org/api/v1"
+    DJANGO_HOMEPAGE_REDIRECT_URL = "https://www.shapeworks-cloud.org"
```

Comment:
You should run `terraform fmt` in the terraform directory, in case there's linting checks on this script.

Reply:
Thanks! Changed in [540ab02](https://github.com/girder/shapeworks-cloud/pull/333/commits/540ab02e09c49cf8828caeb1d4c3ea281ba58f1b)